selfdrive/car/honda/hondacan.py

```diff
@@ -94,14 +100,15 @@ def create_ui_commands(pcm_speed, hud, civic, accord, idx):
     commands.append(make_can_msg(0x39f, msg_0x39f, idx, 0))
   return commands
 
-def create_radar_commands(v_ego, civic, accord, idx):
+def create_radar_commands(v_ego, civic, accord, crv, idx):
   """Creates an iterable of CAN messages for the radar system."""
   commands = []
   v_ego_kph = np.clip(int(round(v_ego * CV.MS_TO_KPH)), 0, 255)
   speed = struct.pack('!B', v_ego_kph)
+
   msg_0x300 = ("\xf9" + speed + "\x8a\xd0" +\
-              ("\x20" if idx == 0 or idx == 3 else "\x00") +\
-                "\x00\x00")
+                  ("\x20" if idx == 0 or idx == 3 else "\x00") +\
```

Comment:
It's better to define the variables to replace such magic numbers(\x20). It can help to improve the readability and encourage more people to contribute to this revolutionary project.